guid_helpers/src/guid_helpers.rs

```diff
@@ -0,0 +1,65 @@
+#![cfg_attr(target_os = "uefi", no_std)]
+
+use r_efi::efi;
+use uuid::uuid;
+
+/// Macro for creating an `efi::Guid` from string representation.
+/// This is a wrapper for `uuid!` from the uuid crate.
+#[macro_export]
+macro_rules! guid {
+    ($guid_str:expr) => {
+        efi::Guid::from_bytes(&uuid!($guid_str).to_bytes_le())
+    };
+}
+
+/// Macro for printing an `efi::Guid` as a string.
+#[macro_export]
+macro_rules! guid_string {

```

Comment:
for what it's worth, this produces a uuid object that implements debug/display, not a string. consider naming it like guid_to_uuid?

Reply:
Would it be more useful to return a uuid that can be printed, or use `format!` to actually return a string?

Reply:
I don't have a strong opinion on that; just wanted to note that your macro is technically not returning a string. I'm okay with it returning a uuid, that gives more options.

Reply:
Added a `guid_to_uuid` macro and updated `guid_string` to return a string